qa/rpc-tests/test_framework/util.py

```diff
@@ -183,6 +187,10 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     if binary is None:
         binary = os.getenv("BITCOIND", "bitcoind")
     args = [ binary, "-datadir="+datadir, "-keypool=1", "-discover=0", "-rest" ]
+    args.extend([
+        '-nuparams=5ba81b19:1', # Overwinter
+        '-nuparams=76b809bb:1', # Sapling
+    ])
```

Comment:
This  seems better than repeating these args at the call sites, but the fact that the exact same code is repeated in this module is question begging.  I don't consider this a blocker.